src/canvas.class.ts

```diff
@@ -1418,6 +1418,11 @@ import { saveObjectTransform } from './util/misc/objectTransforms';
         // this.discardActiveGroup();
         this.discardActiveObject();
         this.clearContext(this.contextTop);
+        if (this._hasITextHandlers) {
+          this.off('mouse:up', this._mouseUpITextHandler);
+          this._iTextInstances = null;
+          this._hasITextHandlers = false;
+        }
```

Comment:
StaticCanvas shouldn't have Itext handlers, if someone come with a real use case we can revert this back